routes/web.php

```diff
@@ -36,4 +36,8 @@
     Route::get('/admin/posts', [\App\Http\Controllers\AdminController::class, 'posts'])->name('admin.posts');
     Route::get('/admin/categories', [\App\Http\Controllers\AdminController::class, 'categories'])->name('admin.categories');
     Route::get('/admin/tags', [\App\Http\Controllers\AdminController::class, 'tags'])->name('admin.tags');
+
+    Route::put('admin/updateUser/{id}', '\App\Http\Controllers\AdminController@updateUser')->name('admin.updateUser');
```

Comment:
route should be uniform code
like this
Route::put('admin/updateUser/{id}', [\App\Http\Controllers\AdminController::class, 'updateUser');

Reply:
also practice to use laravel  dependency injection
ex.
Route::put('admin/updateUser/{user}', [\App\Http\Controllers\AdminController::class, 'updateUser');

Controller
public function updateUser(Request $request, User $user) 
{
   $params = $request->all();

   $user->updater($params);
   if (!$user) {
     // return error
   }

   // return success
}